tools/python/util/android/android.py

```diff
@@ -107,29 +122,49 @@ def start_emulator(
             "4096",
             "-timezone",
             "America/Los_Angeles",
-            "-no-snapshot",
+            "-no-snapstorage",
             "-no-audio",
             "-no-boot-anim",
-            "-no-window",
+            "-gpu",
+            "guest",
+            "-delay-adb",
         ]
+
+        # For Linux CIs we must use "-no-window" otherwise you'll get
```

Comment:
It could because we do not run the agents in interactive mode. https://learn.microsoft.com/en-us/azure/devops/pipelines/agents/agents?view=azure-devops&tabs=yaml%2Cbrowser#interactive-or-service 

I can change that.

Reply:
I used android-arm64-v8a-QNN-crosscompile-ci-pipeline.yml for testing emulator create/start/stop on Linux but that part is commented out due to a QNN issue.

I don't think it's overly important though as we currently use `-no-window` everywhere.